grails-app/services/org/pih/warehouse/core/LocationService.groovy

```diff
@@ -287,18 +291,19 @@ class LocationService {
         }
 
         if (locations) {
-            locations = locations.collect { Location location ->
-                [
-                        id              : location?.id,
-                        name            : location?.name,
-                        foregroundColor : location.fgColor,
-                        backgroundColor : location?.bgColor,
-                        organizationName: location?.organization?.name,
-                        locationType    : location.locationType?.name,
-                        locationGroup   : location?.locationGroup?.name,
-
-                ]
-            }
+            locations = locations.findAll{ Location location -> location.status == LocationStatus.ENABLED }
```

Comment:
Same here, we should probably add a new excludeDisabled boolean argument (with default value = false) to the method.

Reply:
+1